docs/api/utility-process.md

```diff
@@ -1,26 +1,23 @@
-# UtilityProcess
+# utilityProcess
 
-`UtilityProcess` creates a child process with
+`utilityProcess` creates a child process with
 Node.js and Message ports enabled. It provides the equivalent of [`child_process.fork`][] API from Node.js
 but instead uses [Services API][] from Chromium to launch the child process.
 
-`UtilityProcess` is an [EventEmitter][event-emitter].
-
-## Class: UtilityProcess
-
-> Child process with Node.js integration spawned by Chromium.
-
 Process: [Main](../glossary.md#main-process)<br />
 
-### `new UtilityProcess(modulePath[, args][, options])`
+## Methods
+
+### `utilityProcess.fork(modulePath[, args][, options])`
 
-* `modulePath` string - Absolute path to the script that should run as entrypoint in the child process.
+* `modulePath` string - Path to the script that should run as entrypoint in the child process.
 * `args` string[] (optional) - List of string arguments that will be available as `process.argv`
   in the child process.
 * `options` Object (optional)
   * `env` Object (optional) - Environment key-value pairs. Default is `process.env`.
-  * `execArgv` string[] (optional) - List of string arguments passed to the executable. Default is `process.execArgv`.
-  * `stdio` (string[] | string) (optional) - Child's stdout and stderr configuration. Default is `pipe`.
+  * `execArgv` string[] (optional) - List of string arguments passed to the executable.
+  * `cwd` string (optional) - Current working directory of the child process.
+  * `stdio` (string[] | string) (optional) - Child's stdout and stderr configuration. Default is `inherit`.
```

Comment:
Node.js supports [a lot more stuff](https://nodejs.org/api/child_process.html#optionsstdio) here. Is it intentional that we support a much smaller subset of the API? If so, we should probably make that clear in the docs.

Reply:
Since we currently only allow configuring stdout and stderr, I choose the current subset of values since it is the most common usage for these fds. It is possible to support Stream and raw integer values, also Overlapped for windows but I feel it could be a follow-up improvement when adding support to pass arbitrary number of fds to the child process. `ipc` is omitted intentionally since there is always a channel for parent<->child communication using the postMessage API.

Reply:
Refs https://github.com/electron/electron/pull/34980/commits/a82d3223ede85df14b97780ef2bc1cce21e3aa39